test_scripts/API/SetAudioStreamingIndicator/036_ATF_N_TC_SetAudioStreamingIndicator_GENERIC_ERROR_HMI_invalid_audioStreamingIndicator.lua

```diff
@@ -44,7 +44,7 @@ local invalid_data = {
   -- wrong type: integer
   {value = 123, descr = "wrongtype"},
   {value = "TESTING" , descr = "nonexisting_enum"},
-  {value = nil, descr = "missing"}
+  {value = "", descr = "missing"}
```

Comment:
@dboltovskyi, yes idea is for empty, description is updated in f2c513f7048a12e89e02c585effdd7fe2c9c6cbd